web/src/main/ui/app/components/flows-new/edit-flow/ui/new-step-dialog-ui.component.ts

```diff
@@ -28,9 +28,17 @@ export class NewStepDialogUiComponent implements OnInit {
   public stepType: typeof StepType = StepType;
```

Comment:
Since we are not using the ENUM in the HTML anymore, we don't need instantiate stepType. I think this should be part of a separate code clean up task.

Reply:
+1

Reply:
Good point, thanks for bringing attention to it Bruce.